src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/GetUnallocatedCaseService.kt

```diff
@@ -168,6 +169,8 @@ class GetUnallocatedCaseService(
   }
 
   fun getAllCount(): Long = unallocatedCasesRepository.count()
+  fun getUnallocatedCasesTeamCount(teamCodes: List<String>): Flux<UnallocatedCaseTeamCount> = Flux.fromIterable(unallocatedCasesRepository.countByTeamCodes(teamCodes))
```

Comment:
Nots rue about the name - maybe `getUnallocatedCaseCount`? ATM it suggests it's returning a count of teams?!

This comment also applies to the returning data structure's name